static/js/subs.js

```diff
@@ -366,6 +373,11 @@ exports.mark_sub_unsubscribed = function (sub) {
             settings.collapse('hide');
         }
 
+        // Update subscribers count
+        var subscribers_count = subscriber_count_for_sub(sub);
+        var count = stream_data.get_subscribers_count(sub.name) - 1;
+        subscribers_count.text("("+count);
```

Comment:
This seems weird -- why not have the "(" be inside the template and just be setting the number part here?